airflow/runtimes/docker_runtime.go

```diff
@@ -68,15 +68,15 @@ func (rt DockerRuntime) initializeDocker(timeoutSeconds int) error {
 	// If we got an error, Docker is not running, so we attempt to start it.
 	_, err = rt.Engine.Start()
 	if err != nil {
-		return fmt.Errorf(dockerOpenNotice) //nolint:stylecheck
+		return errors.New(dockerOpenNotice)
```

Comment:
nit: we should stay with `fmt.Errorf` I think, since `pkg/errors` is deprecated now, wdyt?

Reply:
I can change this instance from `github.com/pkg/errors` to `errors`?

We might want a separate cleanup PR at some point to remove `github.com/pkg/errors` completely from the CLI, I see many instances of it, and in ways that can't be transparently switched to `errors`.